README.md

```diff
@@ -109,6 +109,28 @@ tree -L 1 ~/code/exercism
 ├── problem-specifications
 └── ruby
 ```
+To explain a bit more, you must follow this commands step-by-step:-
+
+``` bash
+  $ mkdir exercism
+  $ cd exercism
+```
+Fork the [exercism/ruby](https://github.com/exercism/ruby)
+
+Fork the [exercism/problem-specifications](https://github.com/exercism/problem-specifications)
+
+Now you need to clone both the above repositories
```

Comment:
`Now clone these two repositories` maybe, but for sure the `you need to` can go.